lib/workers/repository/dependency-dashboard.ts

```diff
@@ -47,6 +48,24 @@ function checkRebaseAll(issueBody: string): boolean {
   return issueBody.includes(' - [x] <!-- rebase-all-open-prs -->');
 }
 
+function checkCreateConfigMigrationPr(
+  issueBody: string,
+): 'no-checkbox' | 'checked' | 'unchecked' | 'migration-pr-exists' {
+  if (!issueBody.includes('Config Migration needed.')) {
```

Comment:
```suggestion
  if (!issueBody.includes('<!-- create-config-migration-pr -->')) {
```

Reply:
Can't as the html comment is only present incase a pr is needed but not created yet.

Reply:
The string `Config Migration Needed.` (note the period at the end) is already never present:


<img width="551" alt="image" src="https://github.com/user-attachments/assets/695f34cc-ae7d-4f46-be5f-cce0ac1f4427">

---

Please create some anchor comment instead of text and search for it, otherwise it'll break very soon when someone decides to change the text from `Config Migration Needed` to `Config migration needed` or whatever.